src/convertToObject.js

```diff
@@ -12,7 +12,20 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const clearArr = sourceString
+    .split(';')
+    .map(property => property
+      .split(':').map(styleValue => styleValue.trim())
+    )
+    .filter(property => property.length > 1);
+
+  const cssPropertyesObj = {};
+
+  clearArr.map(prop => {
+    cssPropertyesObj[prop[0]] = prop[1];
+  });
```

Comment:
You don't use map results anywhere, so logically it would be better to use general loop or `forEach` here.